pvacseq/lib/vaccine_design.py

```diff
@@ -0,0 +1,378 @@
+#  Input would be a protein fasta
+# ie:
+# python vaccine_design.py test peptides.fa -o output.csv ann  H-2-Kb -e 8
+import shutil
+import sys
+from pathlib import Path
+root = str(Path(__file__).resolve().parents[1])
+sys.path.append(root)
+
+try:
+    from .. import lib
+except ValueError:
+    import lib
+
+import pandas
+import networkx as nx
+import itertools
+from Bio import SeqIO
+import lib
+from parse_output import *
+
+import time
+from simanneal import Annealer
+import random
+import math
+
+
+def define_parser():
+
+    parser = argparse.ArgumentParser('pvacseq vaccine_design')
+    parser.add_argument(
+        "run_name",
+        help="The name of the run being processed." +
+             " This will be used as a prefix for output files."
+    )
+    parser.add_argument('input_file', type=argparse.FileType('r'),
+                        help="Path to input FASTA file")
+    parser.add_argument('method',
+                        choices=PredictionClass.iedb_prediction_methods(),
```

Comment:
Careful when directly copy-pasting.  `PredictionClass` is not imported from anywhere in your code